tests/test_examples/test_configure_hydra.py

```diff
@@ -69,6 +69,20 @@ def test_job_name_with_config_override(tmpdir: Path) -> None:
     assert result == "name_from_config_file"
 
 
+def test_job_override_dirname(tmpdir: Path) -> None:
+    cmd = [
+        "examples/configure_hydra/job_override_dirname/my_app.py",
+        "hydra.sweep.dir=" + str(tmpdir),
+        "lr=0.1,0.01",
+        "optimizer=adam",
+        "seed=999",
+        "-m",
+    ]
+    run_python_script(cmd)
+    assert Path(tmpdir / "lr=0.1,optimizer=adam/seed=999/my_app.log").exists()
+    assert Path(tmpdir / "lr=0.01,optimizer=adam/seed=999/my_app.log").exists()
```

Comment:
Why not directly check that the dir exists instead of a file in it?

Reply:
sure, changed to check dir.